src/buildifier/buildifier_availability.ts

```diff
@@ -33,9 +35,15 @@ const BUILDTOOLS_RELEASES_URL =
  */
 export async function checkBuildifierIsAvailable() {
   const buildifierExecutable = getDefaultBuildifierExecutablePath();
+
   // Check if the program exists (in case it's an actual executable and not
   // an target name starting with `@`).
-  if (!buildifierExecutable.startsWith("@")) {
+  const isTarget = buildifierExecutable.startsWith("@")
+
+  // Check if the program exists as a relative path of the workspace
+  const isRelative = fs.existsSync(path.join(vscode.workspace.workspaceFolders?.[0]?.uri?.fsPath, buildifierExecutable));
```

Comment:
The `existsSync` / `exists` APIs are deprecated in node. Can we switch to `fs.stat` instead?

Reply:
`exists` is deprecated but `existsSync` isn't. I'm happy to change it to the async version though per your other comment 👍🏻